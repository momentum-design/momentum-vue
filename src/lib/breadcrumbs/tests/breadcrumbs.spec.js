import { mount } from '@vue/test-utils';
import Breadcrumbs from '../index.vue';

describe('Breadcrumbs', () => {
  it('should match snapshot', () => {
    const wrapper = mount(Breadcrumbs);
    expect(wrapper.html()).toMatchSnapshot();
  });
});

