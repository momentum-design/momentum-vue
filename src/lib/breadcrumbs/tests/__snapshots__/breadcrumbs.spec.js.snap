// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs should match snapshot 1`] = `<ul class="md-breadcrumbs"></ul>`;
